### ISSUE TITLE
Provide a protocol URI for opening movie info pages.

### ISSUE THREAD

**Issue by phirestalker** [jellyfin/jellyfin-android#694]
**Describe the feature you'd like**
I have ntfy self-hosted and it works great. There is a template that provides a link to the added media item on the Jellyfin server. `{{ServerUrl}}web/index.html#!/details?id={{ItemId}}&serverId={{ServerId}}`. The page apparently exists, but it always makes me log in and then redirects to the main page. This got me thinking about opening the info page in the Android app.

In summary, I want the Android app to accept a URI that will link directly to a media item's page.

**Comment by BotBlake** [comment]
#903 Seems to be a Duplicate of this issue.
Since it contains more details and discussion, I will close THIS ONE as a duplicate.
